.gitlab-ci.yml

```diff
@@ -75,7 +75,7 @@ Validate Code:
   stage: pre-build
   script:
     - cd ${CI_PROJECT_DIR}/
-    - python3 pull-submodules.py --force --dontask --latest:${CI_COMMIT_BRANCH}
+    - python3 pull-submodules.py --force --dontask --latest:${CI_COMMIT_BRANCH} --exclude:iaxo,detector-template
```

Comment:
Shouldn't be included inside lfna?

Reply:
Include what inside lfna?

Reply:
If we do not need them for the pipelines, we better do not pull them

Reply:
Indeed, these are not under `lfna`, what I meant is if would be a good idea to add `projects` option to `pull-submodules.py` following the new projects structure.

Reply:
Not sure if I got exactly what you want

Reply:
Something like
```
- python3 pull-submodules.py --force --dontask --latest:${CI_COMMIT_BRANCH} --projects
```
Otherwise the projects are excluded by default, or the other way around